CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [1.6.3] - 2023-04-12
+
+### Added
+
+- Entry API responses now include the `created_at` field (#3975)
+- `spire-server agent list` CLI command now shows if agents can be re-attested and supports `by_can_reattest` filtering (#3880)
+- `spire-server entry create`, `spire-server entry show` and `spire-server entry update` CLI commands now support hint information, to disambiguate entries with the same SPIFFE ID (#3926, #3787)
+
+### Changed
+
+- X509 CAs minted by an UpstreamAuthority plugin no longer require to have a URI SAN (#3997)
+
+### Fixed
+
+- The `vault` UpstreamAuthority plugin to properly set the URI SAN (#3971)
+- Node resolver data related to deleted nodes is now cleaned (#3873, #3919)
```

Comment:
This could be confused with the (now removed) plugin
```suggestion
- Node selector data related to deleted nodes is now cleaned (#3873, #3919)
```

Reply:
This references one out of two of the PRs for this fix (#3873) as well as the related issue (#3919). I think in most cases we are only referencing the PRs in the changelog.

Shall we change this to reference #3873 and #3941 instead?